src/combine.cpp

```diff
@@ -5847,63 +5848,28 @@ static void handle_oc_property_decl(chunk_t *os)
          // now.
          while (next != nullptr && next->type != CT_PAREN_CLOSE)
          {
-            if (next->type == CT_WORD)
+            if (next->type == CT_OC_PROPERTY_ATTR)
             {
-               if (chunk_is_str(next, "atomic", 6))
+               if (chunk_is_str(next, "atomic", 6) ||
+                   chunk_is_str(next, "nonatomic", 9))
```

Comment:
Not related to this PR but why do we have to pass strlen all the time instead of having a template function ?